lib/Sema/TypeCheckAvailability.cpp

```diff
@@ -42,13 +43,14 @@ using namespace swift;
 ExportContext::ExportContext(DeclContext *DC,
                              AvailabilityContext runningOSVersion,
                              FragileFunctionKind kind,
-                             bool spi, bool exported, bool implicit, bool deprecated,
+                             bool spi, bool exported, bool implicit, bool deprecated, bool ignoresDeprecation,
```

Comment:
I suspect this goes over 80 columns, which is the column limit of our style guidelines. Could you run `git clang-format` over the change?

Reply:
[Ran](https://github.com/apple/swift/pull/59232/commits/440281cc3b5f000f269e918b37c209dbc69f6b3f)